family2family/build.gradle

```diff
@@ -11,13 +10,18 @@ buildscript {
     }
 }
 
+// Update appropriately for releases
+version = '1.0.0-SNAPSHOT'
+
 apply plugin: 'java'
 apply plugin: 'idea'
 apply plugin: 'org.springframework.boot'
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+jar.archiveName = "family2family.${jar.extension}"
```

Comment:
That a result of the build info? Didn't think it was spitting out versions before.